React/game/src/app/screens/Login/components/Login/layout.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+import styles from './styles.scss';
+
+const validate = values => {
```

Comment:
extract this validation function to `src/utils/formValidations.js` file, in order to be reused in another forms

Reply:
👍